pkg/querier/worker/processor_manager.go

```diff
@@ -64,7 +65,9 @@ func (pm *processorManager) concurrency(n int) {
 		n = 0
 	}
 
+	workerId := 0
```

Comment:
i am guessing the workerId does not matter much here. but there is a chance here of re-using workerIds if the concurrency value changes.

Reply:
Good question. I wonder what happens then. Are we restarting the querier?

Reply:
when more schedulers get added? which means each worker gets a smaller concurrency value.
not something that is often done in practice, so it should be alright I think